### ISSUE TITLE
Quality of Life Enhancement: resolve issue of norent items disappear on void, quit, rent

### ISSUE THREAD

**Issue by gonoph** [torilmud/issues#934]
This is a Quality of Life Enhancement where a player would have the ability to void, quit, or rent without destroying any norent items they have in their inventory, equiped, or containers.

In order to maintain the consistency of norent items for game play, I propose that norent items automatically fall the ground when a player voids, quits, or rents. The proposal has several parts:

1. the !rent flag should mask the transient flag (so that !rent items do not poof when dropped).
2. the mud's character save routine would add a block that checks all objects on the character for norent flag.
3. the character save command would display a warning that the character is carrying norent items.
4. upon actual quit/rent/void - any norent items found would be removed from the character (eq, inv, containers/nested) and drop to the ground in the current room.

**Comment by GeneralGormal** [comment]
Yup. Alts are the name of the game and even using your cache as a quick temporary storage can close and you lose stuff.
